src/tasks.js

```diff
@@ -0,0 +1,132 @@
+// Copyright 2019 Google LLC
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     https://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const fetch = require('node-fetch');
+const {v2beta3} = require('@google-cloud/tasks'); // Must be v2beta3 or else tasks creation fails
+const client = new v2beta3.CloudTasksClient();
+
+// Application configuration
+// TODO make the env vars
+const PROJECT = 'serverless-com-demo';
+const QUEUE = 'my-queue';
+const LOCATION = 'us-central1';
+
+// Construct the fully qualified queue name.
+const parent = client.queuePath(PROJECT, LOCATION, QUEUE);
+
+/**
+ * Gets a list of cities.
+ * @returns {string[]} A list of city names.
+ */
+async function getLocationNames() {
+  const URL_LOCATIONS = 'https://api.github.com/gists/7100f98e7d3dd48b3c4d7cb85cfd313f'; // 13k locations
+  // const URL_LOCATIONS = 'https://api.github.com/gists/b503eecba3c49198cc0447550cbe3ccb'; // 19 locations
+  const cities = await fetch(URL_LOCATIONS);
+  const json = await cities.json();
+  const content = json.files['cities.txt'].content;
+  const contentItems = content.split('\n');
+  return contentItems;
+}
+
+/**
+ * Creates a Task Queue
+ * @param {string} queueName The Cloud Tasks queue name.
+ */
+async function createQueue(queueName) {
+  const request = {
+    parent: client.locationPath(PROJECT, LOCATION),
+    queue: {
+      name: client.queuePath(PROJECT, LOCATION, queueName),
+      // rate_limits: {},
```

Comment:
Do you want to keep these commented out?

Reply:
We can uncomment them (and keep them empty).
What do you think?

Since there's no autocompletion for this library, and `_` are unusual in Node objects, I though it would be good to show the user how you could configure these properties.